ament_cmake_python/cmake/ament_python_install_package.cmake

```diff
@@ -13,14 +13,18 @@
 # limitations under the License.
 
 #
-# Install a Python package (and its recursive subpackages).
+# Install a Python package (and its recursive subpackages) as a flat Python .egg
 #
 # :param package_name: the Python package name
 # :type package_name: string
 # :param PACKAGE_DIR: the path to the Python package directory (default:
 #   <package_name> folder relative to the CMAKE_CURRENT_LIST_DIR)
 # :type PACKAGE_DIR: string
-# :param SKIP_COMPILE: if set do not compile the installed package
+# :param VERSION: the Python package version (default: package.xml version)
+# :param VERSION: string
+# :param SETUP_CFG: the path to a setup.cfg file, if provided
```

Comment:
if not provided and `${CMAKE_CURRENT_LIST_DIR}/setup.cfg` exists, that could be taken as the default IMO

Reply:
Yeah, good idea.